best-practices-badge.md

```diff
@@ -0,0 +1,355 @@
+You are standing in an open field of security options, west of a Badge House, with a boarded front door. There is a small website here. Opening the website reveals a small hackmd.
+
+------
+
+Ensuring the security, sustainability, and maintainability of software projects is critically important. Following industry-standard best practices is a great start to doing that, and the [OpenSSF CII Best Practices badge system][bp] is a reliable way to concretely audit and measure how well the project is doing.
```

Comment:
I think this section should be labeled with a heading **Summary** also I would think about adding a who is this document for section.